### PR TITLE
Export zipkin.FromContext for annotating spans inside service implementations

### PR DESCRIPTION
**Pull request by basvanbeek**
By passing context to a service implementation's methods we can pull the existing Zipkin span from context and Annotate items inside our business logic if we export zipkin.FromContext. This will typically be used to time the duration of calls to external resources like databases, caches, etc.

### CODE REVIEW

**Comment by peterbourgon** [comment]
A+! Now that it's an exported method, would you mind adding a quick test?

**Comment by basvanbeek** [comment]
No problem. Will have one in an hour or so.

**Comment by basvanbeek** [comment]
added test and rebased

File: tracing/zipkin/zipkin_test.go [hunk 1]
```diff
@@ -61,6 +61,44 @@ func TestToContext(t *testing.T) {
...
+	if span == nil {
+		t.Fatal("expected a Zipkin span object")
+	}
+	for want, haveFunc := range map[int64]func() int64{
+		traceID:      span.TraceID,
+		spanID:       span.SpanID,
+		parentSpanID: span.ParentSpanID,
+	} {
+		if have := haveFunc(); want != have {
+			name := runtime.FuncForPC(reflect.ValueOf(haveFunc).Pointer()).Name()
```
Comment (peterbourgon) [inline tracing/zipkin/zipkin_test.go]: Ha!

**Comment by peterbourgon** [comment]
Brilliant. Thanks for the contribution!

**Pull request merged**